File: app/src/main/java/com/mxt/anitrend/base/interfaces/event/ActionModeListener.java
package com.mxt.anitrend.base.interfaces.event;

import android.view.ActionMode;
import android.view.ActionMode.Callback;

/**
 * Created by max on 2017/07/17.
 * Action mode trigger callback
 */

public interface ActionModeListener extends Callback {
    void onSelectionChanged(ActionMode actionMode, int count);
}

File: app/src/main/java/com/mxt/anitrend/base/interfaces/event/ActionModeListener.kt
package com.mxt.anitrend.base.interfaces.event

import android.view.ActionMode

/**
 * Created by max on 2017/07/17.
 * Action mode trigger callback
 */
interface ActionModeListener : ActionMode.Callback {
    fun onSelectionChanged(actionMode: ActionMode, count: Int)
}
